torch/testing/_internal/custom_op_db.py

```diff
@@ -0,0 +1,112 @@
+import torch
+from torch.testing._internal.opinfo.core import (
+    OpInfo,
+)
+from torch.testing._internal.common_dtype import all_types_and
+import numpy as np
+from torch._custom_op import CustomOp
+from torch.testing._internal.autograd_function_db import (
+    sample_inputs_numpy_cube,
+    sample_inputs_numpy_mul,
+    sample_inputs_numpy_sort,
+    sample_inputs_numpy_take,
+)
+
+# Note: [custom op db]
+#
+# This is a collection of custom operator test cases written as OpInfos
+# so they can easily be consumed by OpInfo-based tests to check if subsystems
+# support them correctly.
+
+def to_numpy(tensor):
+    return tensor.cpu().numpy()
+
+numpy_cube = CustomOp.define('_torch_testing::numpy_cube(Tensor x) -> (Tensor, Tensor)')
+
+@numpy_cube.impl('cpu')
+@numpy_cube.impl('cuda')
```

Comment:
Huh, weren't you going to support `@numpy_cube.impl('cpu', 'cuda')` syntax?

Minor nit: these examples are a little awkward because, actually, the implementation here would work for ALL devices, not just cpu/cuda.

Reply:
Can do, but it would probably look more like `@numpy_cube.impl('cpu')` or `@numpy_cube.impl(['cpu', 'cuda'])` because I want to leave space for more positional arguments in the future